test/WebSites/BasicWebSite/Views/Home/JsonHelperWithSettingsInView.cshtml

```diff
@@ -3,7 +3,10 @@
 @using Newtonsoft.Json.Serialization
 @{
     var settings = JsonSerializerSettingsProvider.CreateSerializerSettings();
-    settings.ContractResolver = new CamelCasePropertyNamesContractResolver();
+    settings.ContractResolver = new DefaultContractResolver
```

Comment:
You probably don't care since this is a test but caching the camel case and snake case contract resolvers will improve performance.